recipes/libpano13/build.sh

```diff
@@ -0,0 +1,13 @@
+set -ex
+
+./configure \
+    --prefix=${PREFIX} \
+    --with-zlib=${PREFIX} \
+    --with-java=${JAVA_HOME}
+
+cat config.log
+cat Makefile
```

Comment:
These debug statements can be removed/commented.

Reply:
Thanks!